### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by filiptronicek**

### CODE REVIEW

**Comment by NotMoni** [comment]
@filiptronicek Looks good. Can we get some badges and mabe put the banner in there as welll

**Comment by filiptronicek** [comment]
Sure! Any other badges that we should have there?
/cc @yg @bdougie

File: README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Pizza face
+## A Discord bot for the [open-sauced Discord server](https://discord.com/invite/gZMKK5q).
```
Comment (yg) [inline README.md:2]: Add line break

```suggestion
# Pizza face

## A Discord bot for the [open-sauced Discord server](https://discord.com/invite/gZMKK5q).  
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Pizza face
+## A Discord bot for the [open-sauced Discord server](https://discord.com/invite/gZMKK5q).  
+![CI](https://github.com/open-sauced/pizzaface/workflows/Node%20CI/badge.svg)
```
Comment (yg) [inline README.md:3]: Link to workflow page

```suggestion
[![CI](https://github.com/open-sauced/pizzaface/workflows/Node%20CI/badge.svg)](https://github.com/open-sauced/pizzaface/actions?query=workflow%3A%22Node+CI%22)
```

**Comment by NotMoni** [comment]
@filiptronicek @bdougie what license should we use ?

**Comment by filiptronicek** [comment]
I vote for MIT

**Comment by NotMoni** [comment]
Converting this into a draft until we have alll the features in here.

**Comment by bdougie** [comment]
Can we add this too. Emote 3-112

**Comment by NotMoni** [comment]
@filiptronicek Aright can you added the MIT license file, in the pkg.json and in the readme.

Mabe make then seperate PR's

**Comment by filiptronicek** [comment]
Sorry, messed it up with doing it all in this branch :(

File: README.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# Pizza face
+
+![](https://www.dropbox.com/s/mvlrw9wq570qtmu/Emote%203-112.png?dl=1)
```
Comment (yg) [inline README.md:3]: Dropbox link is not working, cuz it is a downloadable link and camo proxy can't handle it!

Changed the URL

```suggestion
![](https://user-images.githubusercontent.com/5713670/85030767-b4526000-b132-11ea-91d4-c3b1e09d1615.png)
```

**Pull request merged**